buyers-guide-locales/en-US/home.properties

```diff
@@ -55,3 +55,7 @@ bg_footer_legal=Legal
 bg_footer_privacy=Privacy
 bg_footer_donate=Donate
 mozilla_blerb2=Mozilla is a global non-profit dedicated to putting you in control of your online experience and shaping the future of the Web for the public good. Visit us at <a href="https://mozilla.org">mozilla.org</a>
+
+#Donation Ask Modal
+donate_modal_title=We all love the web.<br/> Join Mozilla in defending it.
+donate_modal_copy=<p>Let’s protect the world’s largest public resource for future generations.</p><p>Just a few times a year, m<span>&nbsp;</span>ozilla asks for donations. If everyone reading this chipped in, we could wrap this fundraiser in under an hour.</p><p>Together we can keep the web healthy, wonderful and welcoming to all.</p>
```

Comment:
@TheoChevalier What do you think about this string? Should I swap it out into three separate strings, one for each paragraph of this copy?

Reply:
That would be great if you could, yes. I would also take this opportunity to move the `<p>` out of the strings.
However I’m not sure why you need a non-breaking space in mozilla. Is it to prevent Zilla slab from switching it into the logo?

Reply:
OK—once I get updated copy, I'll update this and take out the `p` tags. Yeah, the `&nbsp` is there to prevent the logo from appearing.